packages/swapper/src/types.ts

```diff
@@ -159,6 +160,8 @@ export type GetTradeQuoteInput =
 
 export type EvmSwapperDeps = {
   assertGetEvmChainAdapter: (chainId: ChainId) => EvmChainAdapter
+  fetchIsSmartContractAddressQuery: (userAddress: string, chainId: ChainId) => Promise<boolean>
+  getEthersV5Provider: (chainId: EvmChainId) => ethersV5.providers.JsonRpcProvider
```

Comment:
does look like added back and not needed actually.

Reply:
Very nice catch! This guy now lives in contracts package so doesn't need to be dependency injected. 9a73e2021fdd9b06045d067664dcc3b083fe5f1a